app/models/event.rb

```diff
@@ -6,7 +6,7 @@ class Event < ActiveRecord::Base
   validates :end_date, date: { after_or_equal_to: :start_date }
   validates :organizer_email, confirmation: true, format: { with: /.+@.+\..+/ }, presence: true
   validates :organizer_email_confirmation, presence: true, on: :create
-  validates :website, :code_of_conduct, format: { with: /.+\..+/}
+  validates :website, :code_of_conduct, format: { with: /(http|https):\/\/.+\..+/ }
```

Comment:
The thing is we need the http(s) for showing links (i.e. Link to the code of conduct in app/views/applications/new.html.erb). If we don't make the organizer put it we have to figure out if he has and when not we have to put it in front of whatever he wrote (blah.org) and then we don't know if we should use http or https.